mono/mini/debugger-engine.h

```diff
@@ -207,15 +207,7 @@ typedef struct {
 } ObjRef;
 
 
-static void
-free_objref (gpointer value)
-{
-	ObjRef *o = (ObjRef *)value;
-
-	mono_gchandle_free_internal (o->handle);
-
-	g_free (o);
-}
+void free_objref (gpointer value);
```

Comment:
Please prefix with `mono_debugger_` to "document" that it's a debugger only function.